Chore/document deployment process (#31)
dxw/transition: README.md

Available at /style, the guide documents how transition is using bootstrap, where the app has diverged from default
styles and any custom styles needed to fill in the gaps.

## Deployments

> This service is hosted on dxw's container platform called Dalmation.

Reviewer: ```suggestion
This service is hosted on dxw's container platform called Dalmatian.
```

> Deployments from this applications point of view are done by merging new code into either the develop branch for staging or the master branch for production. Once pushed [DockerHub](https://cloud.docker.com/u/thedxw/repository/docker/thedxw/transition) will build and a new Docker Image.

Reviewer: ```suggestion
Deployments from this application's point of view are done by merging new code into either the develop branch for staging or the master branch for production. Once pushed [DockerHub](https://cloud.docker.com/u/thedxw/repository/docker/thedxw/transition) will build a new Docker Image.
```

### Dalmation

> Once complete. The deployment process to provision this new container hands over to Dalmation.

Reviewer: ```suggestion
Once complete, the deployment process to provision this new container hands over to Dalmatian.
```

This application has a [separate private GitHub repository](https://github.com/dxw/ukri-transition-dalmatian-config) that is responsible for provisioning the required infrastructure. This includes the [Bouncer service](https://github.com/dxw/bouncer) and is done using Terraform.

The way to deploy new containers is manual and involves downtime:

> 1. Within AWS select the dxw-dalmation-1 role

Reviewer: ```suggestion
1. Within AWS select the dxw-dalmatian-1 role
```

1. Visit the ECS service
1. Select the intended cluster (be careful as this cluster is shared)
1. Click 'Tasks'
1. Search by 'transition'
1. Select all tasks running for the app in the intended environment
1. Click 'stop'
1. Those containers will restart and pull the new version of the containers

## Access the console

To access a Rails console or run a rake task on a live environment:

> 1. Within AWS select the dxw-dalmation-1 role

Reviewer: ```suggestion
1. Within AWS select the dxw-dalmatian-1 role
```
